### ISSUE TITLE
Island is not ready (probably deactivated)

### ISSUE THREAD

**Issue by pb-kh** [oasisfeng/island#535]
Every time i try to clone an app to my island, it shows this message mentioned in the title and does nothing.  Even though the work profile is ON, and the island icon in the app is orange

**Comment by mtzirkel** [comment]
I am having the same issue. I just heard about island today but i cannot get it to clone any apps into my Island. I have work profile turned on. I am also getting a message that the shuttle is not ready but this seems to only be when i try to clone gmail.

**Comment by streetbinary** [comment]
Same, when I try to clone an app from mainland, it says "Island is not ready (probably deactivated)".

Strangely, Island app has no permissions and no permissions can be set.

**Comment by pb-kh** [comment]
For me, it was because I uninstalled the app right after creating the island and installed it again

**Comment by jiangnanqw12** [comment]
> Same, when I try to clone an app from mainland, it says "Island is not ready (probably deactivated)".
> 
> Strangely, Island app has no permissions and no permissions can be set.

Same!!

**Comment by arcorogh** [comment]
I get same error message.  I got my new Pixel 9 Pro XL some days ago, and since then Island app throws this error message when I try to clone apps from mainland to island. 
On my Pixel 6 Pro it was working perfectly.

**Comment by J-Bland** [comment]
Interesting. Just got my Pixel 9 Pro Fold and have the same issue.

Can install from Play Store in the Work Profile but cannot do things like Freeze / Unfreeze for instance.

**Comment by ghost** [comment]
Setting up my Pixel 9 Pro XL today and experienced the same.
Tried many way still the same.

Finally use the Play store that's already in the Island to login and install from there and it works.

Hope this help

**Comment by J-Bland** [comment]
Yes the store works though for instance the Contacts app seems to be installed but frozen. I'd ideally like to be able to get this in my work profile.

**Comment by LunkSnee** [comment]
> Yes the store works though for instance the Contacts app seems to be installed but frozen. I'd ideally like to be able to get this in my work profile.

I had the same issue, but I finally fixed this by disabling the Contacts app on Mainland. Then installing/updating Contacts in the Island. Then reenabling Contacts on Mainland. Everything is working!

**Comment by J-Bland** [comment]
> > Yes the store works though for instance the Contacts app seems to be installed but frozen. I'd ideally like to be able to get this in my work profile.
> 
> I had the same issue, but I finally fixed this by disabling the Contacts app on Mainland. Then installing/updating Contacts in the Island. Then reenabling Contacts on Mainland. Everything is working!

I cannot seem to get this to work. Disabled the app in Mainland using Settings, then try to install using Play Store it'll get to 100% and then go back to install. Technically I think it's already in the Island, it's just frozen and I cannot enable it because of this bug.

Are you on a Pixel or some other brand?

**Comment by LunkSnee** [comment]
Pixel 8 Pro. I had to install it from the island, not mainland, after uninstalling and freezing it in mainland.

**Comment by sharkie545** [comment]
> I cannot seem to get this to work. Disabled the app in Mainland using Settings, then try to install using Play Store it'll get to 100% and then go back to install. Technically I think it's already in the Island, it's just frozen and I cannot enable it because of this bug.
> 
> Are you on a Pixel or some other brand?


> Pixel 8 Pro. I had to install it from the island, not mainland, after uninstalling and freezing it in mainland.

I am using a pixel 9 having the same issue. Finally managed to get to work... I basically ended up uninstalling the updates for contacts on mainland from playstore (this gets rid of the updates) then it asked to update right away for contacts. Once it finished it was unfrozen from island works as normal.

**Comment by adizag** [comment]
> > I cannot seem to get this to work. Disabled the app in Mainland using Settings, then try to install using Play Store it'll get to 100% and then go back to install. Technically I think it's already in the Island, it's just frozen and I cannot enable it because of this bug.
> > Are you on a Pixel or some other brand?
> 
> > Pixel 8 Pro. I had to install it from the island, not mainland, after uninstalling and freezing it in mainland.
> 
> I am using a pixel 9 having the same issue. Finally managed to get to work... I basically ended up uninstalling the updates for contacts on mainland from playstore (this gets rid of the updates) then it asked to update right away for contacts. Once it finished it was unfrozen from island works as normal.

That worked for me
Thank!!

**Comment by Sdhb1** [comment]
Is anyone having this issue with samsung galaxy? I am unable to remove Contacts app since it is the built in Samsung app.

**Comment by Abhishmuk** [comment]
> Is anyone having this issue with samsung galaxy? I am unable to remove Contacts app since it is the built in Samsung app.

If it's a built in app, you could try to duplicate via Shizuku (instead of the play store). I was unable to duplicate any app myself, but I had Shizuku set up, and it worked.
